### ISSUE TITLE
The “genesis_time” field is ignored

### ISSUE THREAD

**Issue by KrzysiekJ** [tendermint/tendermint#2294]
When the `genesis_time` field in the `genesis.json` file contains a future date & time, Tendermint ignores that and starts producing blocks right after launch (if there is quorum). An expected behaviour would be to wait until that date.

Tested on 0.21.0.

**Comment by ValarDragon** [comment]
Could you double check if this problem persists on v0.23.0? We've made a couple changes to time, so that may have fixed this problem.

**Comment by KrzysiekJ** [comment]
Yes, it persists.

**Comment by milosevic** [comment]
It should normally be fixed in https://github.com/tendermint/tendermint/pull/2203.

**Comment by ebuchman** [comment]
Don't think this is actually fixed in #2203.

Also not 100% on how we should do this - can we somehow just not start the ConsensusState until genesis time ?

**Comment by melekes** [comment]
> Also not 100% on how we should do this - can we somehow just not start the ConsensusState until genesis time ?

Can't we just `Sleep` in `node#OnStart` somewhere until genesis time is reached?

**Comment by milosevic** [comment]
I was thinking the same.

**Comment by ebuchman** [comment]
> Can't we just Sleep in node#OnStart somewhere until genesis time is reached?

Yes, though I was thinking we'd start up the other services (eg. pex and mempool) ahead of time.

Certainly a simple solution for now is to just sleep.

**Comment by milosevic** [comment]
> Yes, though I was thinking we'd start up the other services (eg. pex and mempool) ahead of time.

This would actually be useful. Lets just check the complexity of doing it. If it's too complex then it might be done postlaunch and we go now with the simple sleep.

**Comment by melekes** [comment]
Simple solution was implemented by @milosevic and merged to develop.
